pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -99,8 +99,8 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 	allNodes := ssn.NodeList
 	predicateFn := func(task *api.TaskInfo, node *api.NodeInfo) ([]*api.Status, error) {
 		// Check for Resource Predicate
-		if ok, reason := task.InitResreq.LessEqualWithReason(node.FutureIdle(), api.Zero); !ok {
-			return nil, api.NewFitError(task, node, reason)
+		if ok, resource := task.InitResreq.LessEqualWithResource(node.FutureIdle(), api.Zero); !ok {
```

Comment:
@Monokaix Hi, I'm kindly asking about the reason or meaning of this PR. After going through all the modifications, I've not getting the meaningful points. It may be better to add descriptions for all PRs.

Reply:
Sorry for delay, I have added the related issus.